src/python/WMCore/WMSpec/Steps/Executors/LogCollect.py

```diff
@@ -63,18 +64,24 @@ def execute(self, emulator=None):
         overrideCatalog = getattr(self.step.application, 'overrideCatalog', None)
 
         overrides = {}
+        params = {}
         if hasattr(self.step, 'override'):
             overrides = self.step.override.dictionary_()
+            if 'siteLocalConfig' in overrides.keys():
+                self.siteCfg = loadSiteLocalConfig()
+                params = self.siteCfg.localStageOut
+            else:
+                params = overrides
 
         # Set wait to over an hour
         waitTime = overrides.get('waitTime', 3600 + (self.step.retryDelay * self.step.retryCount))
 
         # hardcode CERN EOS T2_CH_CERN stageout parameters
         eosStageOutParams = {}
-        eosStageOutParams['command'] = overrides.get('command', "xrdcp")
-        eosStageOutParams['option'] = overrides.get('option', "--wma-disablewriterecovery")
-        eosStageOutParams['phedex-node'] = overrides.get('phedex-node', "T2_CH_CERN")
-        eosStageOutParams['lfn-prefix'] = overrides.get('lfn-prefix', "root://eoscms.cern.ch//eos/cms")
+        eosStageOutParams['command'] = params.get('command', "xrdcp")
+        eosStageOutParams['option'] = params.get('option', "--wma-disablewriterecovery")
+        eosStageOutParams['phedex-node'] = params.get('phedex-node', "T2_CH_CERN")
+        eosStageOutParams['lfn-prefix'] = params.get('lfn-prefix', "root://eoscms.cern.ch//eos/cms")
 
         try:
             eosStageOutMgr = StageOutMgr(**eosStageOutParams)
```

Comment:
Also few more log messages down (at which I cannot comment because are not part of this commit obviously) need to be changed. Those were strictly relying on the fact that the logcollect was actually hardwired to CERN and with this change now they become quite misleading. And all that said, the proper log message may need to be triggered again based on the bool vairable triggering the redirection.